src/prompts/alert/index.tsx

```diff
@@ -0,0 +1,74 @@
+import './alert.scss';
+
+import { Component, h } from 'preact';
+import { PromptUiProps } from "../ui";
+
+export class Alert extends Component<PromptUiProps, never> {
+  onRequestNativePrompt = () => {
+      this.props.onPromptAccepted(this.props.config);
+  };
+
+  onRequestCancel = () => {
+      this.props.onPromptDeclined(this.props.config);
+  };
+
+  render() {
+      const classes = `kumulos-prompt kumulos-prompt-${this.props.promptManagerState} kumulos-alert-container`;
```

Comment:
Do we care about `kumulos-bell-container-${this.props.config.position}` style position classes? Suppose right now there are no other defined positions, but thinking perhaps as we add modal style perhaps we want to target for different transitions.

Can totally defer to a future concern, easier to add than take away.

Reply:
Not at the moment, however will be picking up when impl other prompt types 👍

Reply:
OK, defer until later.